invenio_app_ils/ui/src/invenio_app_ils/routes/components/BackOffice/BackOffice.js

```diff
@@ -1,181 +1,42 @@
-            <Route
-              exact
-              path={BackOfficeRoutes.loansList}
-              component={LoansSearch}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.loanDetails}
-              component={LoanDetailsContainer}
-            />
-            {/* internal locations */}
-            <Route
-              exact
-              path={BackOfficeRoutes.ilocationsCreate}
-              component={InternalLocationEditor}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.ilocationsEdit}
-              component={InternalLocationEditor}
-            />
-            {/* locations */}
-            <Route
-              exact
-              path={BackOfficeRoutes.locationsCreate}
-              component={LocationEditor}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.locationsEdit}
-              component={LocationEditor}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.locationsList}
-              component={LocationList}
-            />
-            {/* patrons */}
-            <Route
-              exact
-              path={BackOfficeRoutes.patronsList}
-              component={PatronsSearch}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.patronDetails}
-              component={PatronDetailsContainer}
-            />
-            {/* series */}
-            <Route
-              exact
-              path={BackOfficeRoutes.seriesList}
-              component={SeriesSearch}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.seriesCreate}
-              component={SeriesEditor}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.seriesEdit}
-              component={SeriesEditor}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.seriesDetails}
-              component={SeriesDetails}
-            />
-            {/* document requests */}
-            <Route
-              exact
-              path={BackOfficeRoutes.documentRequestDetails}
-              component={DocumentRequestDetailsContainer}
-            />
-            <Route
-              exact
-              path={BackOfficeRoutes.documentRequestsList}
-              component={DocumentRequestsSearch}
-            />
-            <Route exact path={BackOfficeRoutes.stats.home} component={Stats} />
-          </Switch>
-        </div>
-      </div>
+        <Grid>
+          {/* Sidebar */}
+          <Grid.Column width={3} style={sidebarStyle}>
+            <Sidebar />
+          </Grid.Column>
+
+          {/* Content */}
+          <Grid.Column
+            stretched
+            width={13}
+            style={contentStyle}
+            verticalAlign={'top'}
+          >
+            <BackOfficeContent />
```

Comment:
just minor naming: I would name it with `BackOfficeRoutingSwitch` or something similar because IMO the component is not having content inside

Reply:
`BackOfficeRoutesSwitch` it is